packages/zcli-apps/src/types.ts

```diff
@@ -97,7 +96,7 @@ export interface Installation {
     id: string;
     plan?: string;
     requirements: {};
-    settings: Settings;
+    settings: Array<any>;
```

Comment:
Can be `Array<Record<string, any>>` like other places and the old interface `Settings` can be removed.